### PR TITLE
Getting started

### PR DESCRIPTION
**Pull request by hsr1991**
added styling to getting started to make website more uniform. all tests pass

### CODE REVIEW

File: ClientApp/src/Pages/GettingStarted/GettingStarted.scss [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
 .getting-started {
     text-align: center;
 }
 
-.selected-icon {
-    padding-left: 5%;
-    padding-right: 5%;
-    background-color: orange;
-    font-size: x-large;
+.btn.btn-outline-primary {
```
Comment (chloejonessoftwire) [inline ClientApp/src/Pages/GettingStarted/GettingStarted.scss:18]: does .btn.btn-outlineprimary want to be indented?

File: ClientApp/src/Pages/GettingStarted/GettingStarted.scss [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
+    background-color: rgb(255, 255, 255);
+    font-size: x-large
 }
-.unselected-icon {
-    padding-left: 5%;
-    padding-right: 5%;
-    font-size: x-large;
+
+.btn.btn-primary {
```
Comment (chloejonessoftwire) [inline ClientApp/src/Pages/GettingStarted/GettingStarted.scss:23]: same here, thing there might be an indentation issue?

File: ClientApp/src/Pages/GettingStarted/GettingStarted.scss [hunk 3]
```diff
@@ -45,13 +45,32 @@
 }
 
 @media only screen and (max-width: 1000px) {
```
Comment (chloejonessoftwire) [inline ClientApp/src/Pages/GettingStarted/GettingStarted.scss]: for these max-width 1000px, and min width 1001px there are constants that yvonne and i used and defined on home.scss you might want to use them too

File: ClientApp/src/Pages/GettingStarted/GettingStarted.test.tsx [hunk 4]
```diff
@@ -11,28 +11,28 @@ test('renders the LocationSummary component without errors', () => {
 test('Onclick Event is working, Canada button works', () => {
       const  { getByTestId  } = render(<GettingStarted />)
       const canadaButton = getByTestId('CanadaButton')
-      expect(canadaButton.className).toBe('unselected-icon')
+      expect(canadaButton.className).toBe('btn btn-outline-primary')
       fireEvent.click(canadaButton)
-      expect(canadaButton.className).toBe('selected-icon')
+      expect(canadaButton.className).toBe('btn btn-primary')
```
Comment (chloejonessoftwire) [inline ClientApp/src/Pages/GettingStarted/GettingStarted.test.tsx]: spare line

File: ClientApp/src/Pages/GettingStarted/GettingStarted.test.tsx [hunk 5]
```diff
@@ -11,28 +11,28 @@ test('renders the LocationSummary component without errors', () => {
...
       const australiaButton = getByTestId('AustraliaButton')
-      expect(scotlandButton.className).toBe('selected-icon')
-      expect(australiaButton.className).toBe('unselected-icon')
+      expect(scotlandButton.className).toBe('btn btn-primary')
+      expect(australiaButton.className).toBe('btn btn-outline-primary')
       fireEvent.click(australiaButton)
-      expect(scotlandButton.className).toBe('unselected-icon')
-      expect(australiaButton.className).toBe('selected-icon')
+      expect(scotlandButton.className).toBe('btn btn-outline-primary')
+      expect(australiaButton.className).toBe('btn btn-primary')
```
Comment (chloejonessoftwire) [inline ClientApp/src/Pages/GettingStarted/GettingStarted.test.tsx]: spare line

**Review by chloejonessoftwire — COMMENTED** [review]
looks good, only given a few code tidy up pointers :)

File: ClientApp/src/Pages/GettingStarted/GettingStarted.scss (hunk 3 above)
Comment (hsr1991) [reply ClientApp/src/Pages/GettingStarted/GettingStarted.scss]: actioned :)

**Review by ChristianMcc — APPROVED** [review]
LGTM! :)

**Pull request merged**